### DISCUSSION TITLE
Is there a javascript version still available?

### DISCUSSION THREAD

**Discussion by packetq** [sourcelair/xterm.js#5064] [Q&A] [answered]
Hi,

Is a javascript version still available for download. If so, what was the last release before switching to typescript?

Thanks,
G

**Comment by jerch** [comment, marked as answer]
Well, all downloadable npm packages are javascript only - the TS stuff gets compiled away before the releases are bundled.

The last javascript only source version was 2.1.0

**Reply by packetq** [reply]
Great! 

Thanks,
G
